--- a/widget/gtk/nsDeviceContextSpecG.cpp
+++ b/widget/gtk/nsDeviceContextSpecG.cpp
@@ -297,7 +297,7 @@ void nsDeviceContextSpecGTK::StartPrintJ
   nsCOMPtr<nsIGIOService> giovfs = do_GetService(NS_GIOSERVICE_CONTRACTID);
   bool shouldUsePortal;
   giovfs->ShouldUseFlatpakPortal(&shouldUsePortal);
-  if (shouldUsePortal) {
+  if (0) {
     GError* error = nullptr;
     GDBusProxy* dbusProxy = g_dbus_proxy_new_for_bus_sync(
         G_BUS_TYPE_SESSION, G_DBUS_PROXY_FLAGS_NONE, nullptr,
@@ -437,7 +437,7 @@ NS_IMETHODIMP nsDeviceContextSpecGTK::En
     bool shouldUsePortal;
     if (giovfs) {
       giovfs->ShouldUseFlatpakPortal(&shouldUsePortal);
-      if (shouldUsePortal) {
+      if (0) {
         // Use the name of the file for printing to match with
         // nsFlatpakPrintPortal
         nsCOMPtr<nsIObserverService> os =
--- a/widget/gtk/nsPrintDialogGTK.cpp
+++ b/widget/gtk/nsPrintDialogGTK.cpp
@@ -910,7 +910,7 @@ nsPrintDialogServiceGTK::Show(nsPIDOMWin
   nsCOMPtr<nsIGIOService> giovfs = do_GetService(NS_GIOSERVICE_CONTRACTID);
   bool shouldUsePortal;
   giovfs->ShouldUseFlatpakPortal(&shouldUsePortal);
-  if (shouldUsePortal && gtk_check_version(3, 22, 0) == nullptr) {
+  if (gtk_check_version(3, 22, 0) == nullptr) {
     nsCOMPtr<nsIWidget> widget = WidgetUtils::DOMWindowToWidget(aParent);
     NS_ASSERTION(widget, "Need a widget for dialog to be modal.");
     GtkWindow* gtkParent = get_gtk_window_for_nsiwidget(widget);
